contribs/src/main/java/com/netflix/conductor/contribs/queue/amqp/AMQPObservableQueue.java

```diff
@@ -536,7 +536,8 @@ protected void receiveMessages() {
 			if (useExchange) {
 				// Consume messages from an exchange
 				getOrCreateExchange();
-				// Create queue if not present based on the settings
+				// Create queue if not present based on the settings provided in the queue URI or configuration properties
+				// Default settings if not provided in the queue URI or properties: isDurable: true, autoDelete: false, isExclusive: false
```

Comment:
Can you put the example url here as to how it is set,
Also mention that this config gets applied to both exchange and queue and we will need a separate key if they need to be separated out

Reply:
Thank you @ritu-p . I have updated the comment.
Can you please check?